client/src/modules/general-ledger/general-ledger.ctrl.js

```diff
@@ -365,6 +199,34 @@ function GeneralLedgerController(
       });
   }
 
+  function renameGidHeaders() {
+    const actions = angular.copy(vm.columns[vm.columns.length - 1]);
```

Comment:
Just FYI, you can also get this value via [`Array.prototype.slice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice).
```js
const actions = vm.columns.slice(-1);
```